### PR TITLE
Add CL2_ prefix to LOAD_TEST_THROUGHPUT

### PR DESCRIPTION
**Pull request by mm4tt**
This is to make it overridable via env

### CODE REVIEW

File: clusterloader2/testing/load/kubemark/500_nodes/override.yaml [hunk 1]
```diff
@@ -1 +1,2 @@
-LOAD_TEST_THROUGHPUT: 50
+CL2_LOAD_TEST_THROUGHPUT: 50
+
```
Comment (wojtek-t) [inline clusterloader2/testing/load/kubemark/500_nodes/override.yaml:2]: nit: remove this empty line

Comment (mm4tt) [reply clusterloader2/testing/load/kubemark/500_nodes/override.yaml]: done, thanks

**Pull request merged**